apps/daily-delight/src/components/self-care-topics-toggle-group.tsx

```
@@ -0,0 +1,17 @@
+import { ToggleGroup, ToggleGroupItem } from '@shared/ui';
+import { FC } from 'react';
+
+interface Props {
+  topics: SelfCareTopic[];
+}
+
+export const SelfCareTopicsToggleGroup: FC<Props> = ({ topics }) => {
+  return (
+    <ToggleGroup type="single">
+      {topics.map((topic) => (
+        <ToggleGroupItem value={topic.id}>{topic.name}</ToggleGroupItem>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)